addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp

```diff
@@ -25,75 +24,62 @@ class baseMan {// Weaponless baseclass
 	headgear[] = {};
 	goggles[] = {"default"};
 	hmd[] = {};
-	// Leave empty to remove all. "Default" > leave original item.
 	faces[] = {};
-	// Leave empty to not change faces.
 	insignias[] = {};
-	// Leave empty to not change insignias
 
-	// All randomized
 	primaryWeapon[] = {};
 	scope[] = {};
 	bipod[] = {};
 	attachment[] = {};
 	silencer[] = {};
-	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
 
-	// Only *Weapons[] arrays are randomized
 	secondaryWeapon[] = {};
 	secondaryAttachments[] = {};
 	sidearmWeapon[] = {};
 	sidearmAttachments[] = {};
-	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
 
-	// These are added to the uniform or vest
 	magazines[] = {};
 	items[] = {
 		 MEDICAL_R ,
 		"ACE_Flashlight_XL50",
-		"ACE_SpraypaintGreen"
+		"ACE_SpraypaintGreen",
+		"ACE_microDAGR"

```

Comment:
MicroDAGR was just about starting to be fielded in 2010: https://gisuser.com/2009/10/rockwell-collins-new-microdagr-gps-receiver-for-soldiers-features-latest-commercial-apps/

Reply:
Fix'd by only giving it to SLs and above.